script/generate-cask.rb

```diff
@@ -7,5 +7,6 @@
 Cask::Cask.each do |c|
   json_filename = "#{c.token}.json"
   IO.write("_data/cask/#{json_filename}", JSON.pretty_generate(c.to_h))
```

Comment:
Could this use `IO.puts` instead? If not, how about doing `#{JSON.pretty_generate(c.to_h)}\n`?

Reply:
Whatever makes the most sense. I was just using my limited ruby knowledge here. XD